src/util.cc

```diff
@@ -48,8 +48,7 @@ TwoByteValue::TwoByteValue(Isolate* isolate, Local<Value> value) {
   const size_t storage = string->Length() + 1;
   AllocateSufficientStorage(storage);
 
-  const int flags =
-      String::NO_NULL_TERMINATION | String::REPLACE_INVALID_UTF8;
+  const int flags = String::NO_NULL_TERMINATION;
```

Comment:
As I've explained in 424722af4541cd0eec1bf299aa8afcdff6284f52, this flag is actually a no-op. `TwoByteValue::TwoByteValue` uses [`v8::String::Write`](https://github.com/nodejs/node/blob/master/src/util.cc#L53) as opposed to [`v8::String::WriteUtf8`](https://github.com/nodejs/node/blob/master/src/util.cc#L26). The flag is [only respected](https://github.com/nodejs/node/blob/736814e064602931615462797c1c12992760993d/deps/v8/src/api.cc#L5622) by `WriteUtf8`, and hence misleading being applied here in `TwoByteValue`.